packages/manager/modules/pci/src/projects/project/kubernetes/service.js

```diff
@@ -108,6 +110,16 @@ export default class Kubernetes {
     });
   }
 
+  getKubeLogKinds(projectId) {
+    return this.iceberg(
+      `/cloud/project/${projectId}/capabilities/kube/log/kind`,
+    )
+      .query()
+      .expand('CachedObjectList-Pages')
```

Comment:
In your other PR you use cursor `CachedObjectList-Cursor`. So try to use the same way to deal with iceberg